kapp-controller.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: kapp-controller
+  version: 0.53.1
+  epoch: 0
+  description: Continuous delivery and package management for Kubernetes
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - git
+      - openssh-client
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 00aa728d6823620c03e3f4917cd565119b17c7d2
+      repository: https://github.com/carvel-dev/kapp-controller
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc@v0.46.0 go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp@v0.46.0 go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc@v1.21.0
+
+  - uses: go/build
+    with:
+      packages: ./cmd/controller
+      output: kapp-controller
+      ldflags: |
+        -X main.Version=v${{package.version}}
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: "upstream image have executable placed at /"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}
+          ln -sf /usr/bin/kapp-controller ${{targets.contextdir}}/kapp-controller
+
+update:
+  enabled: true
+  github:
+    identifier: carvel-dev/kapp-controller
+    strip-prefix: v
+
+test:
+  pipeline:
+    - name: help message
+      runs: |
+        kapp-controller --help
```

Comment:
Can we add another test using the `test/daemon-check-output` pipeline if possible?

Reply:
Thanks for the review @Dentrax, I'm inclined to adding tests using `test/kwok/cluster` pipeline but given there's some problem w.r.t. that pipeline, I've avoided adding the tests for now. Once that's resolved, happy to add the tests here via subsequent commits/patches.